docs/source/v1/WaterCycle/index.rst

```diff
@@ -0,0 +1,45 @@
+E3SMv1 (Water Cycle)
```

Comment:
This page was copied from the original implementation of this PR, #46, from last year. Is this information still accurate?

This page can be seen in rendered form at https://portal.nersc.gov/cfs/e3sm/forsyth/data_docs_59/html/v1/WaterCycle/index.html.

For comparison, here is the [v2 page](https://portal.nersc.gov/cfs/e3sm/forsyth/data_docs_59/html/v2/WaterCycle/index.html).

Reply:
- Add future projections simulations for LR.